phpstan.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+    autoload_files:
+        - tests/bootstrap.php
+    ignoreErrors:
+        - '#Call to an undefined method Cake\\Datasource\\RepositoryInterface::aliasField\(\)#'
+        - '#Call to an undefined method Cake\\ORM\\Table::searchManager\(\)#'
```

Comment:
This should be possible to fix with mixin learning and proper annotation afaik.

Reply:
Yes, but I didn't want to spend time adding a proper phpstan class reflection extension for now.